hr_holidays_public/security/ir.model.access.csv

```diff
@@ -0,0 +1,8 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_hr_holidays_public_user,access_hr_holidays_public,model_hr_holidays_public,base.group_user,1,0,0,0
+access_hr_holidays_public_manager,access_hr_holidays_public,model_hr_holidays_public,hr_holidays.group_hr_holidays_manager,1,1,1,1
+access_hr_holidays_public_line_user,access_hr_holidays_public_line,model_hr_holidays_public_line,base.group_user,1,0,0,0
+access_hr_holidays_public_line_manager,access_hr_holidays_public_line,model_hr_holidays_public_line,hr_holidays.group_hr_holidays_manager,1,1,1,1
+access_hr_holidays_public_manager_next_year_wizard,access_hr_holidays_public_next_year_wizard,model_public_holidays_next_year_wizard,base.group_user,1,1,1,0
+access_hr_holidays_public_manager_user,access_hr_holidays_public,model_hr_holidays_public,base.group_user,1,1,1,0
```

Comment:
Why are you giving such broad permission to all the users? In fact, the first line already grants the needed one.